loki/transformations/data_offload/offload_deepcopy.py

```diff
@@ -424,13 +425,16 @@ class DataOffloadDeepcopyTransformation(Transformation):
     ----------
     mode : str
        Transformation mode, must be either "offload" or "set_pointers".
+    accessor_type : str
```

Comment:
Wouldn't it make sense to also allow a `FieldAPIAccessorType` instance to be provided here?